provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/location/LocationWrapper.java

```diff
@@ -9,10 +9,9 @@
 @Value
 @Builder
 public final class LocationWrapper {
-  @Builder.Default
-  ProviderServicesResponse providerServicesResponse = ProviderServicesResponse.builder().build();
+  ProviderServicesResponse providerServicesResponse;
```

Comment:
I would undo these Changes. They are causing null pointer exceptions/test breakage....

Next sprint, this will be completely refactored anyway as we plan to use NPI in place of PPMS